spec/features/contextual_navigation_spec.rb

```diff
@@ -64,6 +64,27 @@
     and_the_parent_based_breadcrumbs
   end
 
+  scenario "There's a secondary step by step list" do
+    given_theres_a_page_with_a_secondary_step_by_step
+    and_i_visit_that_page
+    then_i_see_the_step_by_step
+    and_the_step_by_step_header
+  end
+
+  scenario "There's between 2-5 secondary step by step lists" do
+    given_theres_are_two_secondary_step_by_step_lists
+    and_i_visit_that_page
+    then_i_just_see_the_step_by_step_related_links
+    and_no_step_by_step_header
```

Comment:
This step isn't needed as the expectation is already included in: `then_i_just_see_the_step_by_step_related_links`